tests/Model/BSONIteratorTest.php

```diff
@@ -19,7 +19,7 @@ class BSONIteratorTest extends TestCase
     #[DataProvider('provideTypeMapOptionsAndExpectedDocuments')]
     public function testValidValues(?array $typeMap, array $expectedDocuments): void
     {
-        $binaryString = implode(array_map(
+        $binaryString = implode('', array_map(
```

Comment:
[`implode()`](https://www.php.net/implode) suggests this alternative syntax is still supported, provided the separator string is empty and named arguments aren't utilized. No objection here, but I'm curious what prompted this change. Was it tooling?

Reply:
That's prompted by rector. I expect the alternative syntax could be deprecated one day.